util/util.go

```diff
@@ -71,3 +76,29 @@ func SetLogger(logToFile bool, logDir string, debug, logJSON bool) error {
 	log15.Root().SetHandler(handler)
 	return nil
 }
+
+// UniqueStrings eliminates duplication from []string
+func UniqueStrings(s []string) []string {
+	if len(s) == 0 {
+		return nil
+	}
+	m := make(map[string]struct{}, len(s))
+	for _, v := range s {
+		m[v] = struct{}{}
+	}
+	uniq := make([]string, 0, len(m))
+	for v := range m {
+		uniq = append(uniq, v)
+	}
+	return uniq
+}
+
+// ParsedOrDefaultTime returns time.Parse(layout, value), or time.Date(1000, time.January, 1, 0, 0, 0, 0, time.UTC) if it failed to parse
+func ParsedOrDefaultTime(layout, value string) time.Time {
```

Comment:
Please apply this function to Debian as well!
https://github.com/maito1201/goval-dictionary/blob/support_fedora/models/debian.go#L44-L45